corporate/views.py

```diff
@@ -131,7 +131,7 @@ def upgrade(
     license_management: Optional[str] = REQ(
         default=None, str_validator=check_string_in(VALID_LICENSE_MANAGEMENT_VALUES)
     ),
-    licenses: Optional[int] = REQ(json_validator=check_int, default=None),
+    licenses: Optional[int] = REQ(converter=to_positive_or_allowed_int(), default=None),
```

Comment:
@timabbott The reason I made this change was because check_int allows non negeative intgers and zereo which are both invalid values for licenses. So I thought  `to_positive_or_allowed_int` is a better fit here. Though this change is not really super necessary since we do check whether the value of licenses are valid in the view.